src/Content/Text/BBCode.php

```diff
@@ -2163,4 +2172,48 @@ public static function getTags($string)
 
 		return $ret;
 	}
+
+	/**
+	 * Replaces mentions in the provided message body for the provided user and network if any
+	 *
+	 * @param $body
+	 * @param $profile_uid
+	 * @param $network
+	 * @return mixed
+	 * @throws \Friendica\Network\HTTPException\InternalServerErrorException
+	 * @throws \ImagickException
+	 */
+	public static function setMentions($body, $profile_uid = 0, $network = '')
```

Comment:
Is `setMentions` only used in that class? Then I would suggest making it private.

Reply:
Seeing https://github.com/friendica/friendica/pull/8729#discussion_r436331984 I believe I will have to use it outside of this class pretty soon.

Reply:
Okay.